src/style.css

```diff
@@ -1,3 +1,125 @@
 body {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-weight: 400;
+  --button-color: rgba(0, 172, 220, 1);
+  --main-text-color: rgba(6, 11, 53, 1);
+  --secondary-text-color: rgba(97, 96, 112, 1);
+}
+
+* {
+  box-sizing: border-box;
   margin: 0;
```

Comment:
it's better to reset it for the list of tags you really need to avoid unexpected behavior

Reply:
I'm not sure if I need to do this because earlier I was told to _move box-sizing: border-box to * selector_...

Reply:
you can leave box-sizing here, but move margin to other selectors, it's inefficient to reset them under this selector